vue-frontend/src/components/home-new/NewPortfolioCTASection.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div
+    class="tw-mt-[20px] lg:tw-mt-[50px] xl:tw-mt-[100px] tw-bg-[#F8F8F8] tw-font-inter-regular"
+  >
+    <div
+      class="tw-container tw-flex tw-flex-col lg:tw-flex-row xl:tw-max-w-[1878px] lg:tw-h-[390px] xl:tw-h-full"
+    >
+      <div
+        class="tw-order-2 lg:tw-order-first lg:tw-w-[50%] xl:tw-w-[40%] 2xl:tw-pl-[30px]"
+      >
+        <img
+          :src="CTAImage[0]"
+          :srcset="`${CTAImage[0]} 400w, ${CTAImage[1]} 800w, ${CTAImage[2]} 1600w, ${CTAImage[3]} 2400w`"
+          alt="portfolio-cta-image"
+          class="-tw-mb-[1rem] sm:-tw-mb-[1.5rem] md:-tw-mb-[2rem] xl:-tw-mb-[1.25rem] lg:-tw-mt-[3.5rem]"
+          loading="lazy"
+        />
+      </div>
+
+      <div class="tw-flex lg:tw-w-[50%]">
+        <div
+          class="tw-m-auto lg:tw-ml-auto xl:tw-m-auto tw-w-[80%] sm:tw-w-[60%] md:tw-w-[50%] lg:tw-w-[90%] 2xl:tw-w-9/12 tw-py-[40px] lg:tw-py-0"
+        >
+          <h1
+            class="tw-text-black-core/[0.87] tw-text-[1.5rem] tw-leading-[1.813rem] md:tw-text-[2.188rem] md:tw-leading-[2.625rem] xl:tw-text-[3.125rem] xl:tw-leading-[3.813rem] tw-font-inter-bold"
+          >
+            Whether you need...
+          </h1>
+          <ul
+            class="tw-pt-6 tw-text-[1rem] tw-text-black-core/[0.6] tw-leading-[1.25rem] md:tw-text-[1.188rem] md:tw-leading-[1.5rem] xl:tw-text-[1.375rem] xl:tw-leading-[1.8125rem] tw-font-inter-semibold"
+          >
+            <li v-for="reason in reasons" :key="reason" class="tw-flex">
+              <span class="tw-pr-1">*</span>
+              <div class="sm:tw-w-[90%]">{{ reason }}</div>
+            </li>
+          </ul>
+          <div
+            class="xl:tw-w-[85%] tw-pt-6 tw-pb-8 tw-text-black-core/[0.6] tw-font-inter-regular tw-text-[0.875rem] tw-leading-[1.125rem] md:tw-text-[1.125rem] md:tw-leading-[1.375rem] xl:tw-text-[1.375rem] xl:tw-leading-[1.813rem]"
```

Comment:
witdh is not required here for large devices, as this paragraph is not looking as figma.